### DISCUSSION TITLE
DXF color exports

### DISCUSSION THREAD

**Discussion by kurt-rhee** [geopandas/geopandas#2720] [Q&A]
Hello, 

Is there a way to export line color information when exporting in DXF format?

```gdf.to_file('test.dxf', driver='DXF')```

**Comment by martinfleis** [comment]
We do not have any specific support for DXF, so you'll have to depend on what GDAL can do. Check the docs https://gdal.org/drivers/vector/dxf.html there. When using `to_file`,  kwargs are passed to fiona and to GDAL, so you should be able to control something. Not sure to which detail though.
